R/plotDispersion.R

```diff
@@ -5,68 +5,154 @@
 #' the plot can instead be Biological Coefficient of Variation (BCV is the square root of
 #' dispersion) against AveLogCPM.
 #'
-#' @param DGEdata Counts matrix or DGEList (Required)
-#' @param designMatrix A design matrix created by stats::model.matrix (Required)
-#' @param plotType Plot type must be canvasXpress or ggplot (Default to canvasXpress).
-#' @param plotCategory One of "dispersion" or "BCV" (Default = "dispersion")
-#' @param symbolSize (Default = 1)
-#' @param symbolShape see
-#'   http://www.cookbook-r.com/Graphs/Shapes_and_line_types/ (Default = 1)
-#' @param symbolColor Default = "darkblue"
-#' @param symbolFill  Default = "darkblue"
-#' @param symbolAlpha Transparency for the points. Value from 0 to 1. Smaller
-#'   indicate more transparency (Default = 0.3)
-#' @param linefitSize (Default = 1)
-#' @param linefitColor (Default = "yellow")
-#' @param lineFit (Default = NULL) Any type supported by geom_smooth (Loess is
-#'   recommended).
-#' @param lineType Any ggplot2 lineType supported by geom_smooth (Default = solid)
-#' @param lineAlpha Alpha desired for geom_smooth line (Default = 1)
+#' @param DGEdata Counts matrix or DGEList (required)
+#' @param designMatrix A design matrix created by stats::model.matrix (required)
+#' @param plotType Plot type must be canvasXpress or ggplot (default = canvasXpress).
+#' @param plotCategory One of "dispersion" or "BCV" (default = "dispersion")
+#' @param symbolSize (default = 6)
+#' @param symbolShape Any supported shape for the points (default = "circle")
+#' \url{http://www.cookbook-r.com/Graphs/Shapes_and_line_types/}
+#' @param symbolColor Fill color (default = "darkblue")
+#' @param symbolTransparency Transparency for the points. Value ranges from 0 to 1. Smaller
+#'   indicate more transparency (default = 0.5)
+#' @param linefitColor (default = "red")
+#' @param lineFit (default = NULL) Any type supported by geom_smooth(if plotType is ggplot) or
+#' one of glm, lm, loess, gam if plotType is canvasXpress. Loess is recommended.
+#'   recommended.
+#' @param lineType Any supported style for the fitLine  (default = solid).
+#'  \url{http://www.cookbook-r.com/Graphs/Shapes_and_line_types/}
 #' @param ... Extra parameters to pass to edgeR::estimateDisp
 #'
 #' @return canvasxpress or ggplot object based on plotType selection
 #'
 #' @examples
 #' \dontrun{
 #'    # canvasxpress plot
-#'    myCxplot <- plotDispersion(myDGElist)
-#'    myCxplot <- plotDispersion(myDGEobj)
+#'    myCxplot <- plotDispersion(myDGElist, designMatrix )
+#'    myCxplot <- plotDispersion(myDGEobj, designMatrix)
```

Comment:
This example uses DGEobj which is not a parameter to the method

Reply:
This is under plotDispersion.R